pxtextension.ts

```diff
@@ -54,14 +63,21 @@ class MlEvent {
   }
 
   //% blockId=mlrunner_is_ml_event
-  //% block="is %this action"
+  //% block="is $this action"
```

Comment:
For my own awareness, what's the difference between `%` and `$` here?

Reply:
I read on, I think on the MakeCode forum, that `%` is the deprecated way to do this, and `&` is now the preferred way. So I've aligned with that.

Reply:
@microbit-carlos I stumbled across the forum post by chance -> https://forum.makecode.com/t/what-is-the-difference-between-use-and-when-defining-parameters-for-blocks/29093

Reply:
That's really useful to know, thanks for sharing!